pkg/setup-hw/dell.go

```diff
@@ -325,7 +325,18 @@ func (dc *dellConfigurator) configBIOS(ctx context.Context) error {
 }
 
 func (dc *dellConfigurator) configPerformance(ctx context.Context) error {
-	return dc.enqueueConfig(ctx, "BIOS.SysProfileSettings.SysProfile", "PerfPerWattOptimizedOs")
+	product, err := lib.DetectProduct()
+	if err != nil {
+		return err
+	}
+	switch product {
+	case lib.R6525, lib.R7525, lib.R7615:
+		return dc.enqueueConfig(ctx, "BIOS.SysProfileSettings.SysProfile", "PerfPerWattOptimizedOs")
+	case lib.R6615: // In 16G server, we cannot disable power saving function even if we set cpu-governor. We need set System Profile to Performance.
+		return dc.enqueueConfig(ctx, "BIOS.SysProfileSettings.SysProfile", "PerfOptimized")
+	default:
+		return fmt.Errorf("unsupported product: %v", product)
```

Comment:
I'm wondering whether we should return an error or not.

In the future, if we forget to update setup-hw for a new model and start up new machines on real data centers,
will the machines be uncontrollable? If so, I want to return nil here with a warning message.

Reply:
Thanks. 
Surely, if setup-hw fails before initial setup of bmc address, it will become uncontrollable(cannot connect iDRAC).
I think it's not good because we need to restart machines by physically pushing power button. So I'll fix this.